src/lib/logger/exceptions.js

```diff
@@ -7,7 +7,9 @@ export const ExceptionCategory = {
   Unexpected: 'unexpected',
 }
 
-export const ExceptionCode = fromPairs(range(1, 13).map(code => [`E${code}`, code]))
+const MAX_EXCEPTION_CODE = 13
+
+export const ExceptionCode = fromPairs(range(1, MAX_EXCEPTION_CODE + 1).map(code => [`E${code}`, code]))
```

Comment:
why not make a map of code->codeString
instead if using findKey which iterates over everything you just get ExceptionCode[code]
@serdiukov-o-nordwhale 
or why not simply codeString = `E${withCode}`

Reply:
this generates enum-like object

ExceptionCode = {
  E1: 1,
  E2: 2,
  // ...
}

which could be easy re-used

Reply:
i've removed findKey and used just `E${code}`